src/App.jsx

```diff
@@ -1,6 +1,12 @@
+import { useState } from 'react';
+import cn from 'classnames';
+
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_NAME = 'name';
```

Comment:
```suggestion
const SORT_FIELDS = {
  length: 'length',
  name: 'name',
};
```
You can create an object for related constants